src/utils.js

```diff
@@ -21,13 +23,21 @@ export function getGoogleVariantFromStyleAndWeight( style, weight ) {
 	return weight + style;
 }
 
+export function getFontStyleLocalized( style ) {
```

Comment:
I think naming this function `localizeFontStyle` works best in this context.

```suggestion
export function localizeFontStyle( style ) {
```